frontend/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM node:10.15.3-stretch
+FROM node:10.16.3-buster
 
 # AWS cli tools
 RUN apt-get update && apt-get install -y \
     build-essential \
     python \
     python-dev \
     python-pip
+RUN pip install PyYAML==5.3.1
```

Comment:
This downgrade fixes the following error. Also see https://github.com/yaml/pyyaml/issues/724.
```
#6 21.25       File "/tmp/pip-build-env-FfX2NQ/lib/python2.7/site-packages/setuptools/command/py36compat.py", line 36, in add_defaults
#6 21.25         self._add_defaults_ext()
#6 21.25       File "/tmp/pip-build-env-FfX2NQ/lib/python2.7/site-packages/setuptools/command/py36compat.py", line 120, in _add_defaults_ext
#6 21.25         self.filelist.extend(build_ext.get_source_files())
#6 21.25       File "/tmp/pip-install-JOgnpr/PyYAML/setup.py", line 201, in get_source_files
#6 21.25         self.cython_sources(ext.sources, ext)
#6 21.25       File "/usr/lib/python2.7/distutils/cmd.py", line 105, in __getattr__
#6 21.25         raise AttributeError, attr
#6 21.25     AttributeError: cython_sources
#6 21.25
#6 21.25     ----------------------------------------
#6 22.13 Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-JOgnpr/PyYAML/
------
executor failed running [/bin/sh -c pip install awscli]: exit code: 1
ERROR: Service 'frontend' failed to build : Build failed
```